.vscode/settings.json

```diff
@@ -1,7 +1,7 @@
 {
   "deno.enable": false,
   "editor.tabSize": 4,
-  "typescript.tsdk": ".yarn/sdks/typescript/lib",
+  "typescript.tsdk": "",
```

Comment:
Why is this removed? We are using the workspace and pnp feature of yarn 3.x. For this yarn uses a patched version of the typescript compiler.